tests/resources/test-types.json

```diff
@@ -778,6 +788,68 @@
                 "forVehicleAxles": null,
                 "defaultTestCode": "wis",
                 "linkedTestCode": null
+              },
+              {
+                "forVehicleType": "psv",
+                "forVehicleSize": "large",
+                "forVehicleConfiguration": "rigid",
+                "forVehicleAxles": null,
+                "defaultTestCode": "wil",
+                "linkedTestCode": null
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "id": "40",
```

Comment:
40 and 41 are already used by hgvs and trailers test-types. you should replace them with 92 and 93

Reply:
Agree, will make changes.